src/components/message/message.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import messageDemo from './demo/default-message.twig';
+import messageWithCloseDemo from './demo/message-with-close.twig';
+import './message.scss';
+
+<!-- @TODO Generate an example of a message with a decorative icon. -->
+
+<Meta title="Components/Message" decorators={[withKnobs]} />
+
+# Message
+
+A containing element intended for short, actionable user messages. Originally
+designed to notify the user of offline network connection status.
+
+<Preview>
+  <Story name="Default" height="120px">
+    {messageDemo({
+      label: text('Message text'),
+      includeClose: boolean('Include close button'),
```

Comment:
Consider making the label the same as the property name so it's a more helpful reference

Reply:
If I've understood this correctly, it the knob should read as follows: 

![Screen Shot 2020-05-04 at 4 20 37 PM](https://user-images.githubusercontent.com/1427548/81022830-82579980-8e23-11ea-82a3-4a68da3823bd.png)

Reply:
I mean if the property is `includeClose`, the label should also be `includeClose`?